config/environments/test.rb

```diff
@@ -25,14 +33,23 @@
   # Disable request forgery protection in test environment.
   config.action_controller.allow_forgery_protection = false
 
-  # Store uploaded files on the local file system in a temporary directory
+  # Store uploaded files on the local file system in a temporary directory.
   config.active_storage.service = :test
 
   # Print deprecation notices to the stderr.
   config.active_support.deprecation = :stderr
 
-  # Raises error for missing translations
-  config.action_view.raise_on_missing_translations = true
+  # Raise exceptions for disallowed deprecations.
+  config.active_support.disallowed_deprecation = :raise
+
+  # Tell Active Support which deprecation messages to disallow.
+  config.active_support.disallowed_deprecation_warnings = []
```

Comment:
Hmm, I would prefer to disallow all deprecation warnings and only allow specific ones if necessary.

Reply:
I've changed this to raise excepts for all deprecations in test and development environment.
I don't think we can allow specific deprecations, but for now we don't need it anyway.